### ISSUE TITLE
Form Builder has issues in Firefox (Mac)

### ISSUE THREAD

**Issue by markpopkes** [wpninjas/ninja-forms#670]
Trying out your plugin for a client and I like all the features, but it has issues in Firefox (currently using v.37). When I go to collapse or expand the form element within the Form Builder, nothing happens. the JS stalls out or something. I checked the JS Console and am getting this error, but it's unrelated to your plugin: NS_ERROR_UNEXPECTED in wp-includes/js/tinymce.min.js line 9. 

Works great in Chrome. Haven't tried Safari yet, but you may want to look into Firefox. Once this error happens, it locks up the form builder and you can do anything! All changes become lost too.
![js-error-ninja-form](https://cloud.githubusercontent.com/assets/999743/7379898/00d30528-edad-11e4-995d-927157113448.png)

**Comment by kstover** [comment]
I just tried to repro this in Firefox 37 on Mac OSX, and I don't get any JS errors. I can edit and save as I'd expect. What OS are you on?

**Comment by markpopkes** [comment]
Thanks for the feedback. I'm on OSX - 10.9.5. (Not quite on Yosemite yet)

**Comment by inspry** [comment]
Second this issue.   I was Using FireFox 37.0.2 then updated to FireFox 38.0.1 on 10.9.5, happened on both versions.  

Seems to only happen on new fields.  

![screen shot 2015-05-15 at 6 32 32 pm](https://cloud.githubusercontent.com/assets/1820634/7662967/376202e2-fb31-11e4-8936-deae183e711a.png)

**Comment by RHKay** [comment]
I'm having a similar issue using Firefox 39 on OSX Yosemite. After adding a few form fields I can no longer expand or collapse any of them, and when I check the error console in Firebug there is a JavaScript error. 

This is the error in the Firebug console:
TypeError: nfFields.get(...) is undefined
...();var t=jQuery(n.target).data("field");nfFields.get(t).toggleMetabox();var i=nf...

I not only experience this issue using Ninja Forms on a clients site in Firefox but I also experienced the same issue using the Ninja Forms demo site in Firefox as well. I thought it may just be an issue with the demo site, but after installing and trying to set up a form on my clients site I'm having the same issue, which is why I enabled the Firebug console to see if there were any errors.
